ngraph/frontend/ir/include/ir_frontend/model.hpp

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <frontend_manager/frontend_manager.hpp>
+#include <inference_engine.hpp>
+#include <ir_frontend/utility.hpp>
+#include <memory>
+#include <ngraph/ngraph.hpp>
+#include <pugixml.hpp>
```

Comment:
pugixml must not be used in public headers. Users will not be able to include such header because pugixml is absent

Reply:
Yes, and IR FE in current state is not supposed to be used by user.

Reply:
fixed in https://github.com/openvinotoolkit/openvino/pull/7249/